### ISSUE TITLE
Wrong type error using mypy

### ISSUE THREAD

**Issue by ArneBachmann** [evhub/coconut#350]
```
from typing import Dict
vcsFolders:Dict[str,str] = {".svn": "svn", ".git": "git", ".bzr": "bzr", ".hg": "hg", ".fslckout": " fossil"}
```
checked with `coconut --target 2 x.coco --mypy` results in
```
x.py:20: error: Dict entry 0 has incompatible type "unicode": "unicode"; expected "str": "str"
x.py:20: error: Dict entry 1 has incompatible type "unicode": "unicode"; expected "str": "str"
x.py:20: error: Dict entry 2 has incompatible type "unicode": "unicode"; expected "str": "str"
x.py:20: error: Dict entry 3 has incompatible type "unicode": "unicode"; expected "str": "str"
x.py:20: error: Dict entry 4 has incompatible type "unicode": "unicode"; expected "str": "str"
```
while
```
from typing import Dict
vcsFolders:Dict[str,str] = {".svn": "svn", ".git": "git", ".bzr": "bzr", ".hg": "hg", ".fslckout": " fossil"}  # type: Dict[str, str]
```
check with `mypy --python-version 2.7 x.py`doesn't.

The question is, where does the unicode come from?

**Comment by evhub** [comment]
@ArneBachmann This was fixed by 1cb8711919fc8bc88a447794005675f5fdfe161c. I've just bumped the develop version in 3bf6352e1a8506d9243b17df98fad37e9f3f7c8a, so once Travis gets a chance to process that commit, you should be able to `pip install -U coconut-develop` and get the fix.

**Comment by ArneBachmann** [comment]
Thanks! For the example above it seems fine, but for my real code I still get
```
sos.py:437: error: Argument 1 to "open" has incompatible type "unicode"; expected "str"
sos.py:437: error: Argument 2 to "open" has incompatible type "unicode"; expected "str"
sos.py:437: error: Argument 3 to "open" has incompatible type "unicode"; expected "str"
sos.py:438: error: Argument 2 to "dump" has incompatible type "StreamReaderWriter"; expected "IO[str]"
sos.py:509: error: Argument 1 to "open" has incompatible type "unicode"; expected "str"
sos.py:509: error: Argument 2 to "open" has incompatible type "unicode"; expected "str"
sos.py:509: error: Argument 3 to "open" has incompatible type "unicode"; expected "str"
sos.py:510: error: Argument 1 to "load" has incompatible type "StreamReaderWriter"; expected "IO[str]"
sos.py:523: error: Argument 1 to "open" has incompatible type "unicode"; expected "str"
sos.py:523: error: Argument 2 to "open" has incompatible type "unicode"; expected "str"
sos.py:523: error: Argument 3 to "open" has incompatible type "unicode"; expected "str"
```
but I'm not able to reproduce this in a small example. Strange thing.
